crates/biome_js_formatter/src/js/lists/parameter_list.rs

```diff
@@ -67,12 +67,22 @@ impl Format<JsFormatContext> for FormatJsAnyParameterList<'_> {
                     FormatTrailingComma::All.trailing_separator(f.options())
                 };
 
+                let has_modifiers = self.list.iter().any(|node| {
+                    matches!(node, Ok(AnyParameter::AnyJsConstructorParameter(
+                        AnyJsConstructorParameter::TsPropertyParameter(parameter),
+                    )) if !parameter.modifiers().is_empty())
```

Comment:
Do we really need t ocheck that the list is not empty? It is already guaren,ted that it is a parameter property?
```suggestion
                    ))
```

Reply:
Indeed, you might be right about that. I will remove this condition.

Reply:
I have removed it.